nanocloud/routes/upload/upload.go

```diff
@@ -195,12 +196,36 @@ func syncUploadedFile(path string) (string, error) {
 		return "", err
 	}
 
-	cmd := exec.Command(filepath.Join(dir, "scripts", "copy.sh"), filepath.Join(dir, path))
-	cmd.Dir = filepath.Join(dir, "scripts")
+	filename := filepath.Join(dir, path)
+
+	winPassword := utils.Env("WIN_PASSWORD", "")
+	winPort := utils.Env("WIN_PORT", "")
+	winUser := utils.Env("WIN_USER", "")
+	winServer := utils.Env("WIN_SERVER", "")
+
+	cmd := exec.Command(
+		"sshpass",
+		"-p",
+		winPassword,
+		"scp",
+		"-o",
+		"StrictHostKeyChecking=no",
```

Comment:
This command needs to ignore knownhost file with this option : "-o UserKnownHostsFile=/dev/null"